webapp/frontend/PHP/classes/Curl.php

```diff
@@ -4,6 +4,96 @@
 
 class Curl
 {
+    public static function generateStudentAnswers($examID) {
+        $exam_storage_user = getenv('EX_STORE_SPLINTER_USER');
+        $exam_storage_password = getenv('EX_STORE_SPLINTER_PASS');
+
+        $ch = curl_init();
+
+        curl_setopt($ch, CURLOPT_URL, "http://splinter_inference_engine:8000/check-exam");
+        curl_setopt($ch, CURLOPT_USERPWD, $exam_storage_user . ":" . $exam_storage_password);
+        curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
+        curl_setopt($ch, CURLOPT_POST, 1);
+        curl_setopt($ch, CURLOPT_POSTFIELDS, '{ "examId": "' . $examID .'", "force":true }');
+        curl_setopt($ch, CURLOPT_HTTPHEADER, array('Content-Type: application/json'));
+        curl_exec($ch);
+        curl_close($ch);
+    }
+
+    public static function generateExamAnswersKeys($examID) {
+
+        $exam_storage_user =  getenv('EX_STORE_SPLINTER_USER');
+        $exam_storage_password = getenv('EX_STORE_SPLINTER_PASS');
+        var_dump("generating".$examID." ".$exam_storage_user." ".$exam_storage_password);
+        $ch = curl_init();
+
+        curl_setopt($ch, CURLOPT_URL, "http://splinter_inference_engine:8000/generate-exam-keys");
+        curl_setopt($ch, CURLOPT_USERPWD, $exam_storage_user . ":" . $exam_storage_password);
+        curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
+        curl_setopt($ch, CURLOPT_POST, 1);
+        curl_setopt($ch, CURLOPT_POSTFIELDS, '{ "examId": "' .  $examID . '", "force":true }');
+        curl_setopt($ch, CURLOPT_HTTPHEADER, array('Content-Type: application/json'));
+        curl_exec($ch);
+        curl_close($ch);
+    }
+
+    public static function getWebdavFileAndUploadSplinter($fileName, $filePath) {
+        $exam_webdav_user =  getenv('EX_STORE_WEBDAV_USER');
+        $exam_webdav_password = getenv('EX_STORE_WEBDAV_PASS');
+
+        // download file from webdav
+        $ch = curl_init();
+
+        $userAgent = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/62.0.3202.94 Safari/537.36';
+
+        curl_setopt($ch, CURLOPT_HEADER, 0);
+        curl_setopt($ch, CURLOPT_VERBOSE, 0);
+        curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
+        curl_setopt($ch, CURLOPT_USERPWD, $exam_webdav_user . ":" . $exam_webdav_password);
+        curl_setopt($ch, CURLOPT_URL, "http://splinter_exam_storage/uploads/".$fileName);
+        curl_setopt($ch, CURLOPT_USERAGENT, $userAgent);
+
+        $output = curl_exec($ch);
+
+        curl_close($ch);
+
+        $fp = fopen($fileName, 'w');
+        fwrite($fp, $output);
+        fclose($fp);
+
+        self::sendFileToSplinter($fileName,$filePath);
```

Comment:
@MateuszKepczynski can we remove selected files from webdav after we copy them to splinter's structures?